document/core/util/macros.def

```diff
@@ -1312,3 +1316,5 @@
 
 .. |error| mathdef:: \xref{appendix/embedding}{embed-error}{\X{error}}
 .. |ERROR| mathdef:: \xref{appendix/embedding}{embed-error}{\K{error}}
+.. |exception| mathdef:: \xref{appendix/embedding}{embed-exception}{\X{exception}}
+.. |EXCEPTION| mathdef:: \xref{appendix/embedding}{embed-exception}{\K{exception}}
```

Comment:
Are these still used?

Reply:
I meant to use these to describe uncaught exceptions but I am removing these two lines for now, as we haven't really discussed uncaught exceptions yet.